action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -464,14 +464,18 @@ func (stateDB *StateDBAdapter) AddLog(evmLog *types.Log) {
 		if len(topics) != 3 {
 			panic("Invalid in contract transfer topics")
 		}
-		from, _ := address.FromBytes(topics[1][12:])
-		to, _ := address.FromBytes(topics[2][12:])
-		stateDB.transactionLogs = append(stateDB.transactionLogs, &action.TransactionLog{
-			Type:      iotextypes.TransactionLogType_IN_CONTRACT_TRANSFER,
-			Sender:    from.String(),
-			Recipient: to.String(),
-			Amount:    new(big.Int).SetBytes(evmLog.Data),
-		})
+		amount := new(big.Int).SetBytes(evmLog.Data)
+		//ignore invalid log
+		if amount.BitLen() != 0 {
```

Comment:
why bit length? what's invalid?

Reply:
Using Bitlen() to judge 0 performs better than Cmp(big.NewInt(0)). Here is Link I found out in the [stack overflow](https://stackoverflow.com/questions/64257065/is-there-another-way-of-testing-if-a-big-int-is-0)
As you mentioned, the word "invalid" might cause misunderstanding. I will change it to "unnecessary transaction log"